cmd/kubeadm/app/constants/constants.go

```diff
@@ -60,6 +64,13 @@ const (
 	// Important: a "v"-prefix shouldn't exist here; semver doesn't allow that
 	MinimumControlPlaneVersion = "1.6.0-alpha.2"
 
+	// Some well-known users and groups in the core Kubernetes authorization system
```

Comment:
I wondered, do we have these easily accessible anywhere else?
Without dependency hell, you know ;)